docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -622,16 +651,19 @@ public class GreetingResourceTest {
 }
 ----
 
-This test code acquires a token using a `password` grant from the test `Auth0` domain which has an application with the client id `test-auth0-client` registered, and which has a user `alice` with a password `alice` created. The test `Auth0` application must have the `password` grant enabled for a test like this one to work. This example code also shows how to pass additional parameters. For `Auth0`, these are the `audience` and `scope` parameters.
-
+This test code acquires a token by using a `password` grant from the test `Auth0` domain, which has registered an application with the client id `test-auth0-client`, and created the user `alice` with password `alice`.
+For a test like this to work, the test `Auth0` application must have the `password` grant enabled.
+This example code also shows how to pass additional parameters.
+For `Auth0`, these are the `audience` and `scope` parameters.
 
 [[integration-testing-keycloak-devservices]]
 ==== Dev Services for Keycloak
 
-Using xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] is recommended for the integration testing against Keycloak.
-`Dev Services for Keycloak` will launch and initialize a test container: it will create a `quarkus` realm, a `quarkus-app` client (`secret` secret) and add `alice` (`admin` and `user` roles) and `bob` (`user` role) users, where all of these properties can be customized.
+Use xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for integration testing against Keycloak.
```

Comment:
```suggestion
Prefer xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for integration testing against Keycloak.
```

`Use` just loses the message IMHO, it makes this option just as a regular alternative.

Reply:
Thanks for your patience with this one @sberyozkin. What about the following slight modification using the "prefer" verb:
"The preferred approach for integration testing against Keycloak is xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak]."

Reply:
@sheilamjones Sure, sounds good